extensions/notebook/src/test/book/book.test.ts

```diff
@@ -3,89 +3,101 @@
-// 	const rootFolderPath = path.join(os.tmpdir(), 'testBook');
-// 	const dataFolderPath = path.join(rootFolderPath, '_data');
-// 	const contentFolderPath = path.join(rootFolderPath, 'content');
-// 	const configFile = path.join(rootFolderPath, '_config.yml');
-// 	const tableOfContentsFile = path.join(dataFolderPath, 'toc.yml');
-// 	const notebookFile = path.join(contentFolderPath, 'notebook.ipynb');
-// 	const markdownFile = path.join(contentFolderPath, 'markdown.md');
-// 	const expectedNotebook = {
-// 		title: 'Notebook',
-// 		url: '/notebook'
-// 	};
-// 	const expectedMarkdown = {
-// 		title: 'Markdown',
-// 		url: '/markdown'
-// 	};
-// 	const expectedExternalLink = {
-// 		title: 'GitHub',
-// 		url: 'https://github.com/',
-// 		external: true
-// 	};
-// 	const expectedBook = {
-// 		sections: [expectedNotebook, expectedMarkdown, expectedExternalLink],
-// 		title: 'Test Book'
-// 	};
+const SEED = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
 
-// 	let mockExtensionContext: TypeMoq.IMock<vscode.ExtensionContext>;
-// 	let bookTreeViewProvider: BookTreeViewProvider;
-// 	let book: BookTreeItem;
+describe('BookTreeViewProvider.getChildren', function (): void {
+	let rootFolderPath: string;
+	const expectedNotebook = {
+		title: 'Notebook',
+		url: '/notebook'
+	};
+	const expectedMarkdown = {
+		title: 'Markdown',
+		url: '/markdown'
+	};
+	const expectedExternalLink = {
+		title: 'GitHub',
+		url: 'https://github.com/',
+		external: true
+	};
+	const expectedBook = {
+		sections: [expectedNotebook, expectedMarkdown, expectedExternalLink],
+		title: 'Test Book'
+	};
 
-// 	this.beforeAll(async () => {
-// 		fs.mkdirSync(rootFolderPath);
-// 		fs.mkdirSync(dataFolderPath);
-// 		fs.mkdirSync(contentFolderPath);
-// 		fs.writeFileSync(configFile, 'title: Test Book');
-// 		fs.writeFileSync(tableOfContentsFile, '- title: Notebook\n  url: /notebook\n- title: Markdown\n  url: /markdown\n- title: GitHub\n  url: https://github.com/\n  external: true');
-// 		fs.writeFileSync(notebookFile, '');
-// 		fs.writeFileSync(markdownFile, '');
-// 		mockExtensionContext = TypeMoq.Mock.ofType<vscode.ExtensionContext>();
-// 		let folder: vscode.WorkspaceFolder = {
-// 			uri: vscode.Uri.parse(rootFolderPath),
-// 			name: '',
-// 			index: 0
-// 		};
-// 		bookTreeViewProvider = new BookTreeViewProvider([folder], mockExtensionContext.object);
-// 	});
+	let mockExtensionContext: TypeMoq.IMock<vscode.ExtensionContext>;
+	let bookTreeViewProvider: BookTreeViewProvider;
+	let book: BookTreeItem;
 
-// 	it('should return all book nodes when element is undefined', async function (): Promise<void> {
-// 		const children = await bookTreeViewProvider.getChildren();
-// 		should(children).be.Array();
-// 		should(children.length).equal(1);
-// 		book = children[0];
-// 		should(book.title).equal(expectedBook.title);
-// 	});
+	this.beforeAll(async () => {
+		try {
+			let res = '';
+			for (let i = 0; i < 8; i++) {
```

Comment:
suggest :

1. Naming this var something like testFolder. res isn't really clear what it's referring to
2. Prefixing the folder name with something like NotebookTestData_ so that it's clear what that folder contains